swarm/src/behaviour.rs

```diff
@@ -224,37 +224,12 @@ pub trait PollParameters {
     /// Iterator returned by [`external_addresses`](PollParameters::external_addresses).
     type ExternalAddressesIter: ExactSizeIterator<Item = AddressRecord>;
 
-    /// Returns the list of protocol the behaviour supports when a remote negotiates a protocol on
-    /// an inbound substream.
-    ///
-    /// The iterator's elements are the ASCII names as reported on the wire.
-    ///
-    /// Note that the list is computed once at initialization and never refreshed.
-    #[deprecated(
-        note = "Use `libp2p_swarm::SupportedProtocols` in your `ConnectionHandler` instead."
-    )]
-    fn supported_protocols(&self) -> Self::SupportedProtocolsIter;
```

Comment:
This was made obsolete only in master and not released yet I think, meaning users have never seen this deprecation warning.

You can cross-check that yourself by looking at the latest version on docs.rs.